### DISCUSSION TITLE
On `yank_main_selection_to_clipboard`, replace with yanked text stops working

### DISCUSSION THREAD

**Discussion by punkbit** [helix-editor/helix#8793] [Troubleshooting]
### Summary

When

```
y = "yank_main_selection_to_clipboard"
```

Pressing R doesn't replace selection with yanked text.

### Reproduction Steps

 

I tried this:

1. open config file
2. set `y = "yank_main_selection_to_clipboard"`
3. reload config
4. use w to select word
5. press y to yank
6. select another word to replace with yanked text
7. pressed shift+r

I expected this to happen:

On step 7 replace selected text with yanked text

Instead, this happened:

On step 7 nothing happens

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

macOS

### Terminal Emulator

kitty 0.27.0

### Installation Method

brew

### Helix Version

helix 23.10 (f6021dd0)

**Comment by Omnikar** [comment]
`R` pastes from the `"` register, not the clipboard. `<space>R` pastes from the clipboard, but based on what you did for `y`, I'd assume you want to rebind `R` to paste from the clipboard, in which case you'd bind it to `replace_selections_with_clipboard`. You would probably also want to rebind `p` and `P`, to `paste_clipboard_after` and `paste_clipboard_before`, respectively.
